chart/templates/deployment.yaml

```diff
@@ -28,32 +28,12 @@ spec:
         args:
         - operator
         {{- if .Values.debug }}
-        - --debug
+        - --v=5
```

Comment:
🤔  does this mean we should replace the "logrus" stuff still around to be able to get debug logs from there?

Reply:
we can do it for other components of the operator